index.html

```diff
@@ -59,8 +58,145 @@ <h1>I'm a Software Developer</h1>
       </li>
       <li><button class="btn"><img src="images/M.png" alt="" /></button></li>
     </ul>
+  </section>
+  <!-- MY projects -->
+  <section class="projects">
 
+    <h1 id="r_w_h">My Recent Works</h1>
+    <hr>
+    <img src="images/Img Placeholder.png" alt="">
+    <div class="container">
+      <h1 id="m">Multi-Post Stories</h1>
+      <p id="description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been
+        the industry's
+        standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+      <div class="box">
+        <ul id="apps">
+          <li>CSS</li>
+          <li>HTML</li>
+          <li>BootStrap</li>
+          <li>Ruby</li>
+        </ul>
+      </div>
+      <button id="btn_1">See Project</button>
+    </div>
+    <!-- Project samples -->
+    <section class="ex1">
+      <div>
+        <h2 class="header-2">Profesional Art Printing Data</h2>
+        <p class="para-2">A daily selection of privately
+          personalized reads; no accounts or sign-ups required. Has
+          been the industry's standard.</p>
+        <ul class="apps-1">
+          <li class="i">HTML</li>
+          <li class="i">BootStrap</li>
+          <li class="i">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn-5">
+        See Project
+      </button>
+    </section>
+    <section class="ex1">
+      <div>
+        <h2 class="header-2">Profesional Art Printing Data</h2>
+        <p class="para-2">A daily selection of privately
+          personalized reads; no accounts or sign-ups required. Has
+          been the industry's standard.</p>
+        <ul class="apps-1">
+          <li class="i">HTML</li>
+          <li class="i">BootStrap</li>
+          <li class="i">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn-5">
+        See Project
+      </button>
+    </section>
+    <section class="ex1">
+      <div>
+        <h2 class="header-2">Profesional Art Printing Data</h2>
+        <p class="para-2">A daily selection of privately
+          personalized reads; no accounts or sign-ups required. Has
+          been the industry's standard.</p>
+        <ul class="apps-1">
+          <li class="i">HTML</li>
+          <li class="i">BootStrap</li>
+          <li class="i">Ruby</li>
+        </ul>
+      </div>
+      <button class="btn-5">
+        See Project
+      </button>
```

Comment:
- In the design template, there are a total of six project cards in this section, I can only see three in your project. Please fix this 🙏. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
- Also, you have not yet implemented the button interactions styles for all your portfolio buttons. Please fix this 👍 👍 
Button interactions:
![gwrgwrgw](https://github.com/Rdzaman20000/Portfolio/assets/73607512/5b5f06b6-2055-417b-bfd3-8ebc229fc7f0)